mbed_tools_lib/python_helpers.py

```diff
@@ -0,0 +1,23 @@
+"""Helpers for python language related functions."""
+import sys
+from collections import namedtuple
+from typing import NamedTuple, Iterable, Any, cast
+
+
+def is_python_more_recent(major: int, minor: int) -> bool:
```

Comment:
Could we change the name of this to something like `minimum_python_version`?  `is_python_more_recent` is a bit confusing to me. Even `is_python_more_recent_than(3,7)` makes more sense to the caller imo.

Reply:
ok will do